Dockerfile

```diff
@@ -1,48 +1,45 @@
 # ==============================
-FROM helsinkitest/python:3.9-slim as appbase
+FROM registry.access.redhat.com/ubi8/python-39 as appbase
 # ==============================
+
+USER 0
```

Comment:
Using magic numbers is not a good idea IMO, how about using the name of the user?

```bash
docker compose up --build
docker exec -it kukkuu-backend id 0
```
->
```
uid=0(root) gid=0(root) groups=0(root)
```
i.e. `root`

Reply:
this is changed
`USER root`